snuba/clickhouse/translators/snuba/mappers.py

```diff
@@ -24,6 +24,7 @@
     String,
 )
 
+
```

Comment:
💀

Reply:
I had touched that file (but removed the changes) but black autoformatted it anyways.  If I delete this line, black will put it right back